src/group/GroupCard.js

```diff
@@ -49,7 +51,6 @@ export default class GroupCard extends PureComponent<Props, State> {
     const { selected } = this.state;
 
     const user = users.find(x => x.email === email);
-
```

Comment:
Make sure not to do unintended changes.

Reply:
Thanks for the feedback. I will make that this will not repeat again. I am new to sending PR that's why but now I am okay with it, so I will avoid such from happening again.